src/enemyc.cpp

```diff
@@ -642,176 +642,43 @@ void KEnemy::LoadEnemies(const std::string& fullPath, Raster* enemy_gfx)
-        iss >> fighter_loaded_from_disk.mmp;
-        // Defeat Item Probability: chance of finding any items after defeat
-        iss >> fighter_loaded_from_disk.dip;
-        // Defeat Item Common: item found commonly of the time
-        iss >> fighter_loaded_from_disk.defeat_item_common;
-        // Defeat Item Rare: item found rarely
-        iss >> fighter_loaded_from_disk.defeat_item_rare;
-        // Steal Item Common: item found commonly from stealing
-        iss >> fighter_loaded_from_disk.steal_item_common;
-        // Steal Item Rare: item found rarely when stealing
-        iss >> fighter_loaded_from_disk.steal_item_rare;
-        // Enemy's strength (agility & vitality set to zero)
-        iss >> fighter_loaded_from_disk.stats[eStat::Strength];
-        fighter_loaded_from_disk.stats[eStat::Agility] = 0;
-        fighter_loaded_from_disk.stats[eStat::Vitality] = 0;
-        // Intelligence & Sagacity (both the same)
-        iss >> fighter_loaded_from_disk.stats[eStat::Intellect];
-        fighter_loaded_from_disk.stats[eStat::Sagacity] = fighter_loaded_from_disk.stats[eStat::Intellect];
-        // Defense against: Speed, Spirit, Attack, Hit, Defense, Evade, Magic (in that order)
-        iss >> fighter_loaded_from_disk.stats[eStat::Speed];
-        iss >> fighter_loaded_from_disk.stats[eStat::Aura];
-        iss >> fighter_loaded_from_disk.stats[eStat::Spirit];
-        iss >> fighter_loaded_from_disk.stats[eStat::Attack];
-        iss >> fighter_loaded_from_disk.stats[eStat::Hit];
-        iss >> fighter_loaded_from_disk.stats[eStat::Defense];
-        iss >> fighter_loaded_from_disk.stats[eStat::Evade];
-        iss >> fighter_loaded_from_disk.stats[eStat::MagicDefense];
-        // Bonus
-        iss >> fighter_loaded_from_disk.bonus;
-        fighter_loaded_from_disk.bstat = 0;
-        // Current weapon type
-        iss >> fighter_loaded_from_disk.current_weapon_type;
-        // Weapon elemental type
-        iss >> fighter_loaded_from_disk.welem;
-        // Undead Level (defense against Undead attacks)
-        iss >> fighter_loaded_from_disk.unl;
-        // Critical attacks
-        iss >> fighter_loaded_from_disk.crit;
-        // Temp Sag & Int for Imbued
-        iss >> fighter_loaded_from_disk.imb_s;
-        // Imbued stat type (Spd, Spi, Att, Hit, Def, Evd, Mag)
-        iss >> fighter_loaded_from_disk.imb_a;
-
-        fighter_loaded_from_disk.img =
-            std::make_shared<Raster>(fighter_loaded_from_disk.cw, fighter_loaded_from_disk.cl);
-        enemy_gfx->blitTo(fighter_loaded_from_disk.img.get(), imagefile_x_coord, imagefile_y_coord, 0, 0,
-                          fighter_loaded_from_disk.cw, fighter_loaded_from_disk.cl);
-        iss >> fighter_loaded_from_disk.imb[0];
-        iss >> fighter_loaded_from_disk.imb[1];
+        fighter.img = std::make_shared<Raster>(fighter.cw, fighter.cl);
+        enemy_gfx->blitTo(fighter.img.get(), fighter.cx, fighter.cy, 0, 0, fighter.cw, fighter.cl);
```

Comment:
One of the *could-be-significant* changes made here, was that `KFighter::cx` and `KFighter::cy` were never set before, and the positions within the source image were read into temporary variables and only used in this `enemy_gfx->blitTo()` call.

Those `cx` and `cy` values are actually used throughout the game as temporaries, such as when the enemy's attack stance changes or animates.

So I figured that, as they're always being written to by those other places in code before they're being used, whatever values were here would just get clobbered anyway, so we might as well use them to temporarily store those values.